tests/mock_crust/utils.rs

```diff
@@ -872,7 +873,10 @@ fn add_node_to_section<T: Rng>(
     nodes.iter_mut().for_each(|node| {
         node.inner.set_next_relocation_dst(Some(relocation_name));
         node.inner
-            .set_next_relocation_interval(Some((prefix.lower_bound(), prefix.upper_bound())));
+            .set_next_relocation_interval(Some(XorTargetInterval(
+                prefix.lower_bound(),
+                prefix.upper_bound(),
+            )));
```

Comment:
Seems like it might be worth adding a `From<Prefix>` impl for XorTargetInterval for convenience.

Reply:
Only used a few times, and later commits in that PR add an inclusive_range() function to prefix that is more generally used.